augur/support/filter/subsample.py

```diff
@@ -58,7 +58,7 @@ def get_groups_for_subsampling(strains, metadata, group_by=None):
     >>> get_groups_for_subsampling(strains, metadata, group_by)
     Traceback (most recent call last):
       ...
-    augur.filter.FilterException: The specified group-by categories (['missing_column']) were not found.
+    augur.support.filter.errors.FilterException: The specified group-by categories (['missing_column']) were not found.
```

Comment:
I think it might be preferable to maintain the exception class availability via `augur.filter` by re-exporting `FilterException` and others from there? e.g. adding a line like `from augur.support.filter.errors import FilterException, …` to `augur/filter.py`?

Reply:
Done: d63d23d00a96e3a0e1294ec9f28b7ee11b3fe3f6

But the doctest traceback here still shows `augur.filter.errors.FilterException` which I think is reasonable.